src/includes/server/http/response/AResponse.hpp

```diff
@@ -0,0 +1,28 @@
+//
+// Created by pmerku on 11/03/2021.
+//
+
+#ifndef ARESPONSE_HPP
+#define ARESPONSE_HPP
+
+#include <map>
+#include <string>
+#include "server/http/HTTPClient.hpp"
+
+namespace response {
+
+	class AResponse {
+	private:
+		std::string _response;
+
+	public:
+		AResponse();
+		virtual ~AResponse();
+
+		const std::string &getResponse() const;
+		void setResponse(const NotApache::HTTPClient &client);
+	};
+
+} // namespace response
+
+#endif //ARESPONSE_HPP
```

Comment:
why is this called AResponse? its not an abstract class.

Reply:
and it seems kinda pointless, the builder is supposed to be used directly in HTTPResponder::generateResponse()

Reply:
I was thinking of making a general response class and then have a concrete class for HTTP and one for the Terminal. But right now yes it's pointless